src/Storefront/Framework/Routing/Router.php

```diff
@@ -14,9 +13,6 @@
 
 class Router implements RouterInterface, RequestMatcherInterface, WarmableInterface, ServiceSubscriberInterface
```

Comment:
this class only calls the decorated one. so it is obsolete?

Reply:
yes it is, maybe for second pull request